package.json

```diff
@@ -592,6 +607,12 @@
       }
     ],
     "menus": {
+      "webview/context": [
+        {
+          "command": "SAS.saveHTML",
+          "when": "webviewSection == 'resultsView'"
```

Comment:
Looking at VS Code's doc, `webviewSection` is for different sections in a webview, which we don't need.
I think we should primarily check `webviewId`, which is the first parameter passed to `window.createWebviewPanel`.
It is currently set to "SASSession", perhaps we can revise it to something like "SASResult".

Reply:
Done https://github.com/sassoftware/vscode-sas-extension/pull/550/commits/7834fea3dd0fa8e2b567e73b159125587d079ba9